.github/workflows/main.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     if: >-
       (github.ref_name == 'main' ||
         github.ref_name == 'development' ||
+        github.ref_name == 'fix/deep-binding-spa1738' ||
```

Comment:
Please make sure you remove this before merging.

Reply:
DONE